src/node/blockstorage.cpp

```diff
@@ -893,6 +894,6 @@ void ThreadImport(ChainstateManager& chainman, std::vector<fs::path> vImportFile
             return;
         }
     } // End scope of CImportingNow
-    chainman.ActiveChainstate().LoadMempool(args);
+    chainman.ActiveChainstate().LoadMempool(mempool_path);
```

Comment:
c4249c1243981f42091950d1295c2be6bd4bf118: I think you probably wanted to call `ShouldPersistMempool(args) ? MempoolPath(args) : fs::path{}` here? Which also makes more sense to me, as `args` is already available in this scope.

Reply:
I guess that depends on whether we want `ThreadImport` to be part of `libbitcoinkernel` or not. I'm operating under the assumption that we do want it to be part of `libbitcoinkernel`, so whenever there's an opportunity to untangle it from `ArgsManager`, I take it.